src/components/integrated/OakLessonInfoCard/OakLessonInfoCard.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+
+import {
+  OakFlex,
+  OakHeading,
+  OakHeadingTag,
+  OakIcon,
+  OakIconName,
+} from "@/components/base";
+import { OakHandDrawnCard, OakPrimaryInvertedButton } from "@/components/ui";
+
+export type LessonInfoCardProps = {
+  iconName: OakIconName;
+  infoCardTitle: string;
+  infoCardDescription: string | string[];
+  tag: OakHeadingTag;
+  downloadLabel?: string;
+  onDownloadClick?: (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+  ) => void;
+};
+
+export const OakLessonInfoCard = (props: LessonInfoCardProps) => {
+  const {
+    iconName,
+    infoCardTitle,
+    infoCardDescription,
+    tag,
+    downloadLabel,
+    onDownloadClick,
+    ...rest
+  } = props;
+
+  return (
+    <OakFlex
+      $borderRadius={"border-radius-l"}
+      $pa={"inner-padding-xl"}
+      $flexDirection={"column"}
+      $gap={"space-between-s"}
+      $background={"white"}
+      {...rest}
+    >
+      <OakFlex $alignItems={"center"} $gap={"space-between-ssx"}>
+        <OakIcon iconName={iconName} />
+        <OakHeading $font={"heading-6"} tag={tag}>
+          {infoCardTitle}
+        </OakHeading>
+      </OakFlex>
+      <OakFlex
+        $font={"text-primary"}
+        $flexDirection={"column"}
+        $gap={"space-between-m"}
+      >
+        {Array.isArray(infoCardDescription) ? (
+          infoCardDescription.map((description) => {
+            return <OakFlex>{description}</OakFlex>;
+          })
+        ) : (
+          <OakFlex>{infoCardDescription}</OakFlex>
+        )}
+        {onDownloadClick && downloadLabel && (
+          <OakFlex $justifyContent={"flex-end"}>
+            <OakPrimaryInvertedButton
+              onClick={onDownloadClick}
+              iconName="download"
+              isTrailingIcon
+              $font={"heading-7"}
+            >
+              {downloadLabel}
+            </OakPrimaryInvertedButton>
+          </OakFlex>
+        )}
+      </OakFlex>
+    </OakFlex>
+  );
+};
+
+export const OakStaticMessageCard = (
+  props: Omit<LessonInfoCardProps, "onDownloadClick">,
+) => {
+  const {
+    iconName,
+    infoCardTitle,
+    infoCardDescription,
+    tag,
+    downloadLabel,
+    ...rest
+  } = props;
+
+  return (
+    <OakHandDrawnCard {...rest} fill={"bg-decorative2-subdued"}>
+      <OakFlex
+        $pa={"inner-padding-none"}
+        $flexDirection={"column"}
+        $gap={"space-between-s"}
+      >
+        {" "}
+        <OakFlex $alignItems={"center"} $gap={"space-between-ssx"}>
+          <OakIcon iconName={iconName} />
+          <OakHeading $font={"heading-6"} tag={tag}>
+            {infoCardTitle}
+          </OakHeading>
+        </OakFlex>
```

Comment:
Personally I see this as another component altogether since it has use for all cards (and it's just copied from `OakLessonInfoCard` above. If you did that it would remove the ambiguous naming of the props coming into this component and simplify `OakLessonInfoCard`.

In OWA it could be used like:

```jsx
<OakStaticMessageCard>
  <OakCardHeader tag="h1" iconName="question-mark">Header title</OakCardHeader>
  {Array.isArray(infoCardDescription) ? (
    infoCardDescription.map((description) => {
      return <OakFlex>{description}</OakFlex>;
    })
  ) : (
    <OakFlex>{infoCardDescription}</OakFlex>
  )}
</OakStaticMessageCard>
```

At which point `OakStaticMessageCard` is pretty redundant and you could use `OakHandDrawnCard` directly in OWA

```jsx
<OakHandDrawnCard fill="bg-decorative2-subdued">
  <OakCardHeader tag="h1" iconName="question-mark">Header title</OakCardHeader>
  {Array.isArray(infoCardDescription) ? (
    infoCardDescription.map((description) => {
      return <OakFlex>{description}</OakFlex>;
    })
  ) : (
    <OakFlex>{infoCardDescription}</OakFlex>
  )}
</OakHandDrawnCard>
```

If you then added an example to the stories for `OakHandDrawnCard` with the header I think you'd be good.

Reply:
I like this idea; so ive exported the custom cards and header from the oakLessonIfoCard folder, and demod them in storybook